packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/embedEmojisInto..st

```diff
@@ -0,0 +1,17 @@
+utility
+embedEmojisInto: aString
+
+	| reencoded runs |
+	
+	reencoded := (aString convertToEncoding: 'UTF-16') convertFromEncoding: 'UTF-16'.
```

Comment:
What is the purpose of reencoding the String?

Reply:
Squeak stores its Strings in the UTF-16 format. This line provides an easy way to convert the String from UTF-16 to Unicode.
e. g. as string with the following character values: `55357 56836` gets turned into `128516` which is now a single character, whose hexadecimal codepoint now directly leads to the file for this emoji: 😄

Reply:
Ok cool, maybe a comment for the next group would be great here?